RuriLib/Functions/Crypto/Crypto.cs

```diff
@@ -626,18 +626,58 @@ private static byte[] PerformCryptography(byte[] data, ICryptoTransform cryptoTr
         #region JWT
         public static string JwtEncode(JwtAlgorithmName algorithmName, string secret, IDictionary<string, object> extraHeaders, IDictionary<string, object> payload)
         {
-            IJwtAlgorithm algorithm = algorithmName switch
+            IJwtAlgorithm algorithm = null;
+            RSA rsa = null;
```

Comment:
There is a potential memory leak since `RSA` implements `IDisposable` but it's never disposed. Maybe it would make sense to wrap all the code below in a try/finally statement like

```cs
try
{
    switch ...
}
finally
{
    rsa?.Dispose();
}
```

Please test after making this change to make sure RSA is not disposed too early.